README.md

```diff
@@ -3,14 +3,615 @@
 Elastic Search engine
 
 ###Install:
+
+##### Cli script based approach
 ```
 sudo php vendor/oat-sa/lib-tao-elasticsearch/bin/activateElasticSearch.php <pathToTaoRoot> <host> <port> <login> <password>
 ```
  - `pathToTaoRoot` it's root path of your tao
- - `host` it's host of your elasticsearch environment. `localhost` by defaut.
+ - `host` it's host of your elasticsearch environment. `localhost` by default.
  - `port` it's port of your elasticsearch environment. `9200` by default.
  - `login` it's login for you elasticsearch environment. Optional property.
  - `password` it's password for you elasticsearch environment. Optional property.
+ 
+##### Seed file based approach
```

Comment:
This seed seems very big and difficult to maintain, could we have an alternative by using this seed keys and/or a path to a file?

Reply:
I think we should keep only variable related to elastic search connection.

Reply:
Yep, trying to implement something like @gabrielfs7 suggested here https://github.com/oat-sa/lib-tao-elasticsearch/pull/37#discussion_r507962683